templates/definition/meter/growatt.yaml

```diff
@@ -0,0 +1,85 @@
+template: growatt
```

Comment:
growatt-hybrid

analog auch Dateiname

Reply:
Ja, ich muss noch herausfinden für welche Inverter dieses Registerset genau gilt. In der Protokollbeschreibung steht dazu nämlich genau nichts und es gibt zwei verschiedene Sets...

Reply:
Habs umbenannt- jetzt passt erstmal die Description zum Dateinamen.